src/core/Microsoft.PowerToys.Settings.UI.Lib/stylecop.json

```diff
@@ -0,0 +1,21 @@
+﻿{
```

Comment:
I think we should use single `stylecop.json` that is [located here](https://github.com/microsoft/PowerToys/blob/master/src/codeAnalysis/StyleCop.json) rather than a copy around projects.